libs/blocks/graybox-promote/components/graybox-promote-lit.js

```diff
@@ -49,8 +49,8 @@ const getJson = async (url, errMsg = `Failed to fetch ${url}`) => {
   return sheet;
 };
 
-const getSheetValue = (data, key) =>
-  data?.find((obj) => obj.key?.toLowerCase() === key?.toLowerCase())?.value;
+// eslint-disable-next-line
+const getSheetValue = (data, key) => data?.find((obj) => obj.key?.toLowerCase() === key?.toLowerCase())?.value;
```

Comment:
I'm guessing the lint issue is around line length? If so, maybe split it up?

```suggestion
const getSheetValue = (data, key) => data?.find(
      (obj) => obj.key?.toLowerCase() === key?.toLowerCase())?.value;
```

Reply:
True. The issue was with the length. I have splited it up.